changelog/README.md

```diff
@@ -9,6 +9,7 @@
 - [documentation] JAVA-2416: Update paging section in the manual
 - [improvement] JAVA-2402: Add setTracing(boolean) to StatementBuilder
 - [bug] JAVA-2466: Set idempotence to null in BatchStatement.newInstance
+- [improvement] JAVA-2452: Allow "none" as a compression option
```

Comment:
Nit: changelog entries should be inserted first in their section for new drivers, and last for legacy drivers. I know, we don't make it any easier :-D